Apps/Src/Voltage.c

```diff
@@ -200,17 +219,26 @@ SafetyStatus Voltage_OpenWire(void){
     
     RTOS_BPS_MutexPend(&MinionsASIC_Mutex, OS_OPT_PEND_BLOCKING);
     
+    //int32_t openModules = LTC6811_run_openwire_multi(NUM_MINIONS, Minions, false);
     LTC6811_run_openwire_multi(NUM_MINIONS, Minions, false);
 
+    /*
+    To do: fix this to work with VOLT_TAP_DIST array
```

Comment:
bro has a todo in a pr. (i know it says draft pr)

Reply:
Open wire is turned off and we don't need it for static monitoring, but there was no comment saying it was wrong.